search/package.json

```diff
@@ -10,47 +10,62 @@
     "search": "AWS_SDK_LOAD_CONFIG=true npx babel-node --presets @babel/env src/main/nodejs/search_client.js"
   },
   "dependencies": {
-    "@react-native-community/async-storage": "^1.11.0",
-    "archiver": "^5.3.0",
-    "aws-appsync": "^4.0.0",
-    "aws-xray-sdk": "^3.0.1",
+    "@aws-crypto/sha256-js": "^5.2.0",
+    "@aws-sdk/client-batch": "^3.751.0",
+    "@aws-sdk/client-cognito-identity-provider": "^3.750.0",
+    "@aws-sdk/client-cloudwatch-logs": "^3.750.0",
+    "@aws-sdk/client-dynamodb": "^3.751.0",
+    "@aws-sdk/client-lambda": "^3.750.0",
+    "@aws-sdk/client-s3": "^3.750.0",
+    "@aws-sdk/client-sfn": "^3.750.0",
+    "@aws-sdk/client-sts": "^3.750.0",
+    "@aws-sdk/credential-providers": "^3.750.0",
+    "@aws-sdk/credential-provider-node": "^3.750.0",
+    "@aws-sdk/signature-v4": "^3.374.0",
+    "@aws-sdk/lib-dynamodb": "^3.751.0",
+    "@aws-sdk/lib-storage": "^3.750.0",
+    "@smithy/protocol-http": "^5.0.1",
+    "aws-amplify": "^5.3.15",
+    "aws-appsync-auth-link": "^3.0.7",
+    "archiver": "^7.0.1",
     "exponential-backoff": "^3.1.0",
-    "geotiff": "^1.0.8",
-    "graphql": "^15.3.0",
-    "graphql-tag": "^2.10.4",
-    "isomorphic-fetch": "^2.2.1",
-    "jimp": "^0.16.1",
-    "mathjs": "^7.2.0",
-    "moment": "^2.25.3",
-    "react": "16.8.3",
```

Comment:
I don't know why we had react in here because we weren't actually using it so I removed it.